.well-known/dat

```diff
@@ -0,0 +1,2 @@
+dat://1eab91e668e98e4588d578ebf4d42b1e315bec57488c3411749704c021915783/
```

Comment:
How do you get `1eab91e668e98e4588d578ebf4d42b1e315bec57488c3411749704c021915783 `?

Reply:
It's a generated when the archive is first shared. https://github.com/datproject/dat#sharing-data